business/core/tests/user_test.go

```diff
@@ -127,6 +129,27 @@ func userCrud(t *testing.T) {
 		t.Errorf("Should be able to see updates to Department")
 	}
 
+	if len(saved.Roles) != len(upd.Roles) {
+		t.Logf("got: %v", saved.Roles)
+		t.Logf("exp: %v", upd.Roles)
+		t.Errorf("Should be able to see updates to Roles")
+	} else {
```

Comment:
I HATE else clauses.  I am going to code this myself.  THIS WAS A GOOD FIND. Can't believe that has been missing.

Reply:
Since we are only working with a single role in this slice, this will work fine.

```
if !saved.Roles[0].Equal(upd.Roles[0]) {
	t.Logf("got: %v", saved.Roles[0].Name())
	t.Logf("exp: %v", upd.Roles[0].Name())
	t.Errorf("Should be able to see updates to Role")
}
```